core/state/statedb_test.go

```diff
@@ -1170,12 +838,16 @@ func TestGenerateMultiCoinAccounts(t *testing.T) {
 
 	// Get latest snapshot and make sure it has the correct account and storage
 	snap := snaps.Snapshot(root)
-	snapAccount, err := snap.Account(addrHash)
+	snapAccount, err := snap.AccountRLP(addrHash)
```

Comment:
Since there are no accessors for Extra on the SlimAccount, the only way to access it is to get the RLP and convert it to a full account (StateAccount) which we can call `types.IsMuliCoin` on

Reply:
https://github.com/ava-labs/libevm/pull/79 adds the accessor to `SlimAccount`.

Reply:
We can switch to this in a separate PR